docs/pypi.md

```diff
@@ -33,10 +33,39 @@ finished loading:
 
 ```
 def do_work(*args):
-  import snowballstemmer
-  snowballstemmer.stemmer('english')
-  stemmer.stemWords('go goes going gone'.split())
+    import snowballstemmer
+    stemmer = snowballstemmer.stemmer('english')
+    print(stemmer.stemWords('go goes going gone'.split()))
 
 import micropip
 micropip.install('snowballstemmer').then(do_work)
```

Comment:
This example is then a bit redundant with the one below.

Reply:
Fair point.

Reply:
I'll let someone else delete this one if that's okay?